Assets/Scripts/Interactable/InteractableObject.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InteractableObject : MonoBehaviour, IInteractableObject
```

Comment:
I like the idea of this class, but with this now we can remove the interface (as the implementation doesn't change), and if we want to add interactable objects that have nothing to do with mini games (like role-playing stuff, or even picking up trash) we'll have to create a separate class for those types of interactables (also rename this one to something more suiting) but maybe we can create a base class for this one, see what they have in common

Reply:
I agree. For now I removed the interface, but we should definitely have a base class for all interactable objects